hyperon_das/client.py

```diff
@@ -32,110 +28,50 @@ def _send_request(self, payload) -> str | dict | int:
         except requests.exceptions.RequestException as e:
             raise e
 
-    def get_node(
-        self,
-        node_type: str,
-        node_name: str,
-        output_format: QueryOutputFormat = QueryOutputFormat.HANDLE,
-    ) -> Union[str, Dict]:
+    def get_node_handle(self, node_type: str, node_name: str) -> str:
         payload = {
-            'action': 'get_node',
-            'input': {
-                'node_type': node_type,
-                'node_name': node_name,
-                'output_format': output_format.name,
-            },
+            'action': 'get_node_handle',
+            'input': {'node_type': node_type, 'node_name': node_name},
         }
         return self._send_request(payload)
 
-    def get_nodes(
-        self,
-        node_type: str,
-        node_name: str = None,
-        output_format: QueryOutputFormat = QueryOutputFormat.HANDLE,
-    ) -> Union[List[str], List[Dict]]:
+    def get_link_handle(self, link_type: str, link_targets: List[str]) -> str:
```

Comment:
This is wrong. We don't need to delegate `get_link_handle()` to the remote DAS. This function shouldn't be here. Node and links handles can be calculated using a static method in the basic AtomDB. Se my design for reference.